packages/@react-stately/collections/docs/collections.mdx

```diff
@@ -338,37 +340,76 @@ let [items, setItems] = useState([
 </Tree>
 ```
 
+-->
+
 ## Asynchronous loading
 
 Many collection components support loading data asynchronously. This is supported by passing an `isLoading` prop,
 which is `true` while the data is loading. Once the data is loaded, `isLoading` should be set to `false`, and the
 `items` prop should be set to the loaded data.
 
-TODO: useAsyncList/useAsyncTree
+The <TypeLink links={dataDocs.links} type={dataDocs.exports.useAsyncList} /> hook from `@react-stately/data` can be
+used to manage async data loading from an API. Pass a `load` function to `useAsyncList`, which returns the items
+to render. You can use whatever data fetching library you want, or the builtin
```

Comment:
builtin -> built-in
https://www.dictionary.com/browse/built-in
https://www.merriam-webster.com/dictionary/built-in